### PR TITLE
simple form and conrtoller logic for adding new board

### PR DESCRIPTION
**Pull request by meganjstreet**

### CODE REVIEW

File: app/controllers/boards_controller.rb [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
   end
 
   def index
+  end
+
+  def create
+    @board = Board.new(board_params)
+
+    if @board.save
+      redirect_to @board, notice: 'Board was successfully created.'
```
Comment (KIKMAKER) [inline app/controllers/boards_controller.rb:18]: maybe I'm old fashioned but I prefer board_path(@board)

File: app/views/boards/new.html.erb [hunk 2]
```diff
@@ -1,2 +1,7 @@
-<h1>Boards#new</h1>
-<p>Find me in app/views/boards/new.html.erb</p>
+<h1>New Board</h1>
+<%= simple_form_for @board do |f| %>
+  <%= f.input :name %>
+  <%= f.input :game %>
+  <%= f.input :friendship_id, collection: @friendships, label_method: lambda { |friendship| friendship.user == current_user ? friendship.friend_user.username : friendship.user.username} %>
```
Comment (KIKMAKER) [inline app/views/boards/new.html.erb:5]: took me a while to figure out why this was necessary!

**Review by KIKMAKER — APPROVED** [review]

**Comment by KIKMAKER** [comment]
Mega work Mr Kit-T

File: app/views/boards/new.html.erb (hunk 2 above)
Comment (meganjstreet) [reply app/views/boards/new.html.erb:5]: I will work on leaving comments and explainations for things :)

**Pull request merged**